### PR TITLE
Additional handedness by EEPROM examples

### PR DESCRIPTION
**Pull request by filterpaper**
## Description

* Improve bootloader parameter clarity with a table
* Add explicit examples for `qmk flash` commands 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by waffle87 — APPROVED** [review]

File: docs/feature_split_keyboard.md [hunk 1]
```diff
@@ -132,25 +132,22 @@ To enable this method, add the following to your `config.h` file:
...
-
-Example:
+|Microcontroller Type|Bootloader Parameter|
+|--------------------|--------------------|
+|AVR controllers with Caterina bootloader<br>(e.g. Pro Micro)|`:avrdude-split-left`<br>`:avrdude-split-right`|
+|AVR controllers with the stock Amtel DFU or DFU compatible bootloader<br>(e.g. Elite-C)|`:dfu-split-left`<br>`:dfu-split-right`|
+|ARM controllers with a DFU compatible bootloader<br>(e.g. Proton-C)|`:dfu-util-split-left`<br>`dfu-util-split-right`|
+|ARM controllers with a UF2 compatible bootloader<br>(e.g. RP2040)|`:uf2-split-left`<br>`:uf2-split-right`|
 
+Example for `crkbd/rev1` keyboard with normal AVR Promicro MCUs, reset the left controller and run:
```
Comment (waffle87) [inline docs/feature_split_keyboard.md:144]:
```suggestion
Example for `crkbd/rev1` keyboard with normal AVR Pro Micro MCUs, reset the left controller and run:
```

**Review by daskygit — CHANGES_REQUESTED** [review]
Could you switch the examples and options over to `qmk flash` please? There's a requirement for `qmk` to be working and it's less prone to user error e.g. running `make` from a different directory.

**Comment by filterpaper** [comment]
> Could you switch the examples and options over to `qmk flash` please? There's a requirement for `qmk` to be working and it's less prone to user error e.g. running `make` from a different directory.

Updated with `qmk` commands

**Comment by keyboard-magpie** [comment]
As alluded to on the low profile server- do we need to document something about the eeprom reset steps that seem to be needed for EE_hands on emulated eeprom MCUs?

**Comment by filterpaper** [comment]
> As alluded to on the low profile server- do we need to document something about the eeprom reset steps that seem to be needed for EE_hands on emulated eeprom MCUs?

Do you mean clearing EEPROM prior to flashing `uf2-split-*`?

**Comment by keyboard-magpie** [comment]
> 

Yes- although I've not been able to replicate the issues users on LPK and QMK servers have with this. Input from Dasky will be useful.

**Comment by sigprof** [comment]
> Do you mean clearing EEPROM prior to flashing `uf2-split-*`?

This should not be needed after #18352 (included in 0.19.0).

**Comment by keyboard-magpie** [comment]
So probably worth adding a note about ensuring their local QMK repository is up to date.

File: docs/feature_split_keyboard.md [hunk 2]
```diff
@@ -132,28 +132,23 @@ To enable this method, add the following to your `config.h` file:
 #define EE_HANDS
 ```
 
-Next, you will have to flash the EEPROM files once for the correct hand to the controller on each halve. You can do this manually with the following bootloader targets while flashing the firmware:
+Next, you will have to flash the EEPROM files once for the correct hand to the controller on each halve. You can do this manually with the following bootloader targets using `qmk flash -kb <keyboard> -km <keymap> -bl <bootloader>` command to flash:
```
Comment (daskygit) [inline docs/feature_split_keyboard.md:135]: I was going to submit a PR to deal with some issues after this merged but looks like we might as well deal with this here.

> Next, you will have to flash the EEPROM files once

This doesn't apply to all controller and bootloader combinations.

Blackpill + tinyuf2, can be flashed once and it'll retain it's handedness in emulated eeprom when flashed again with a non-handed firmware.

Blackpill + dfu, has to be flashed every time with handed firmware as the emulated eeprom section is at the start of flash and overwritten.

I'm undecided if flash once should just be removed for simplicity or expanded.

```suggestion
Next, you will have to flash the correct handedness option to the controller on each halve. You can do this manually with the following bootloader targets using `qmk flash -kb <keyboard> -km <keymap> -bl <bootloader>` command to flash:
```
Maybe a note later?
```
?> Some controllers may not retain the handedness option when being flashed again, in this case you'll need to always flash each half with the correct bootloader target from above.
```
could probably be worded better.

Comment (filterpaper) [reply docs/feature_split_keyboard.md]: Added as an exception footnote to avoid wordiness in the previous paragraph.

**Review by daskygit — APPROVED** [review]

**Pull request merged**